### ISSUE TITLE
Package request: zluda-lshqqytiger

### ISSUE THREAD

**Issue by Pandapip1** [NixOS/nixpkgs#312502]
**Project description**

An actively-maintained fork of ZLUDA.

**Metadata**

* homepage URL: https://github.com/lshqqytiger/ZLUDA
* source URL: https://github.com/lshqqytiger/ZLUDA
* license: apache-2.0
* platforms: unix, linux

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by mweinelt** [comment]
> This branch is [122 commits ahead of](https://github.com/lshqqytiger/ZLUDA/compare/vosen%3AZLUDA%3Amaster...master), [187 commits behind](https://github.com/lshqqytiger/ZLUDA/compare/master...vosen%3AZLUDA%3Amaster) vosen/ZLUDA:master.

I don't think that panned out.

**Comment by Pandapip1** [comment]
> I don't think that panned out.

Fair enough, but both are still actively maintained and I do believe lshqqytiger's fork has better compatibility.
